roles/reproducer/tasks/prepare_networking.yml

```diff
@@ -152,6 +152,45 @@
       server=/{{ _domain }}/127.0.0.2
       server=/apps-crc.testing/127.0.0.2
       server=/crc.testing/127.0.0.2
+      server=/utility/127.0.0.2
+
+- name: Feed utility records
+  block:
+    - name: Ensure "utility" domain is local
+      become: true
+      notify: "Restart dnsmasq"
+      ansible.builtin.copy:
+        dest: "/etc/cifmw-dnsmasq.d/utility.conf"
+        content: |-
+          local=/utility/
+
+    # Using ctlplane gateway IP should ensure we're targeting an IP
+    # on the hypervisor, reachable by the VMs, ensuring it can discuss
+    # with the host (vbmc and sushy_emulator reaching libvirt, etc).
+    - name: Build utility records dataset
+      vars:
+        _target_net: "{{ cifmw_networking_env_definition.networks.ctlplane }}"
+        _controller_net: >-
+          {{
+            cifmw_networking_env_definition.instances['controller-0'].networks
+          }}
+        cifmw_dnsmasq_host_record:
+          - names:
+              - "ipmi.utility"
+              - "{{ inventory_hostname }}.utility"
+            ips:
+              - "{{ _target_net.gw_v4 | default('') }}"
+              - "{{ _target_net.gw_v6 | default('') }}"
+            state: present
+          - names:
+              - "sushy.utility"
```

Comment:
this should work - though how can we actually test it? @hjensas maybe?

Reply:
A bit blocked by that unbound issue in the nat64 right now.
This looks good, if it works for IPv4 it should work for IPv6 - I think we can merge this as is and refine IPv6 support if it does not just work.